.travis.yml

```diff
@@ -21,6 +23,11 @@ install:
   - ./scripts/travis-django
   - pip install -r requirements-dev.txt
   - pip freeze
+  - curl -sL https://deb.nodesource.com/setup_10.x | sudo bash -
```

Comment:
I would like to just add the steps needed for ubuntu trusty instead of this. it's probably just a `add-ppa` command, or whatever it's called, and one `apt-get update`